### ISSUE TITLE
Error packing APK

### ISSUE THREAD

**Issue by StePPBD** [kefir500/apk-editor-studio#18]
Hi,
can't really figure out how to continue.
Downloaded the APK, edited only the MIN SDK Version number and tried to save.

This is the error log:

> I: Using Apktool 2.4.1
> I: Copying C:\Users\Stefan\AppData\Local\Temp\apk-editor-studio\apk\{b2863aec-bedc-41de-84bd-2cea16aa32a1} classes.dex file...
> I: Building resources...W: C:\Users\Stefan\AppData\Local\Temp\apk-editor-studio\apk\{b2863aec-bedc-41de-84bd-2cea16aa32a1}\res\values\styles.xml:13: error: Resource entry AppTheme already has bag item android:statusBarColor.
> W: C:\Users\Stefan\AppData\Local\Temp\apk-editor-studio\apk\{b2863aec-bedc-41de-84bd-2cea16aa32a1}\res\values\styles.xml:12: Originally defined here.
> W: 
> brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [C:\Users\Stefan\AppData\Local\Temp\brut_util_Jar_48184101202116821.tmp, p, --forced-package-id, 127, --min-sdk-version, 21, --target-sdk-version, 28, --version-code, 20201014, --version-name, 3.1.5, --no-version-vectors, -F, C:\Users\Stefan\AppData\Local\Temp\APKTOOL6716385221402879324.tmp, -e, C:\Users\Stefan\AppData\Local\Temp\APKTOOL4901568594843723801.tmp, -0, arsc, -I, C:\Users\Stefan\AppData\Local\apk-editor-studio\frameworks\1.apk, -S, C:\Users\Stefan\AppData\Local\Temp\apk-editor-studio\apk\{b2863aec-bedc-41de-84bd-2cea16aa32a1}\res, -M, C:\Users\Stefan\AppData\Local\Temp\apk-editor-studio\apk\{b2863aec-bedc-41de-84bd-2cea16aa32a1}\AndroidManifest.xml]

Any ideas?

**Comment by kefir500** [comment]
Hi,
Could you please provide (a link to) the APK?

**Comment by Bfaschat** [comment]
Use apktool v2.4.0 and problem will be solved

**Comment by StePPBD** [comment]
Hi,

I managed to resolve the issue.
First, I downloaded latest apktool (issue persisted).
Then I cleared the framework directory and used the apktool with the option --use-aapt2.
This seems to do the trick, building the app worked.
I then signed it manually and tadaa - it works.

Thank you very much for your support!

Btw, the app is called "Lesen lernen" bei eKidz.eu and required Android 6. The kids' tablet is only running 5.1., revealing the reason why I needed the tool.
